libs/testdiff/replacement.go

```diff
@@ -208,3 +209,20 @@ func PrepareReplacementsDevVersion(t testutil.TestingT, r *ReplacementsContext)
 	t.Helper()
 	r.append(devVersionRegex, "[DEV_VERSION]")
 }
+
+func PrepareReplacementSdkVersion(t testutil.TestingT, r *ReplacementsContext) {
+	t.Helper()
+	r.Set(databricks.Version(), "[SDK_VERSION]")
+}
+
+func goVersion() string {
+	gv := runtime.Version()
+	ssv := strings.ReplaceAll(gv, "go", "v")
+	sv := semver.Canonical(ssv)
+	return strings.TrimPrefix(sv, "v")
+}
+
+func PrepareReplacementsGoVersion(t testutil.TestingT, r *ReplacementsContext) {
+	t.Helper()
+	r.Set(goVersion(), "[GO_VERSION]")
```

Comment:
Strings like "1.23.6" are not very unique and may show up elsewhere. You can avoid false matches by capturing a bigger string:

r.Set("go/" + goVersion(), "go/[GO_VERSION]")

Reply:
Should we keep this as is and handle that case when it comes up? I'd presume that we'll not run into this case for a long time if ever.

There are not a lot of acceptance tests where we print integers.

Reply:
yes, it was general advice, not call to action. We can deal with this when we see it's problem.